go.mod

```diff
@@ -18,3 +18,5 @@ require (
 	google.golang.org/grpc v1.21.1
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 )
+
+go 1.13
```

Comment:
@hoshir is this okay? It keeps getting added (probably because I'm using 1.13 on my machine).

Reply:
Developers will need go version equal or above 1.13, but I think it's OK.

Reply:
Okay, I've updated the main README to say 1.13 is required instead of 1.11beta2.